x-pack/plugins/ml/public/application/routing/routes/new_job/from_lens.tsx

```diff
@@ -22,17 +19,19 @@ export const fromLensRouteFactory = (): MlRoute => ({
   breadcrumbs: [],
 });
 
-const PageWrapper: FC<PageProps> = ({ location, deps }) => {
+const PageWrapper: FC<PageProps> = ({ location }) => {
   const { lensId, vis, from, to, query, filters, layerIndex }: Record<string, any> = parse(
     location.search,
     {
       sort: false,
     }
   );
 
-  const { context } = useResolver(undefined, undefined, deps.config, deps.dataViewsContract, {
+  // TODO check license and capabilities
```

Comment:
Reminder to remove this once this has been tested.

Reply:
removed in https://github.com/elastic/kibana/pull/159176/commits/71565c45c0401e2d44be5d03a934f8c34c1403f1